applications/main/nfc/plugins/supported_cards/trt.c

```diff
@@ -40,7 +40,10 @@ static bool trt_parse(const NfcDevice* device, FuriString* parsed_data) {
         const uint8_t* full_record_pointer = &data->page[FULL_SALE_TIME_STAMP_PAGE].data[0];
         uint32_t latest_sale_record = bit_lib_get_bits_32(partial_record_pointer, 3, 20);
         uint32_t latest_sale_full_record = bit_lib_get_bits_32(full_record_pointer, 0, 27);
-        if(latest_sale_record != (latest_sale_full_record & 0xFFFFF)) break;
+        if(latest_sale_record != (latest_sale_full_record & 0xFFFFF))
+            break; // check if the copy matches
+        if((latest_sale_record == 0) | (latest_sale_full_record == 0))
```

Comment:
why `|` ?

Reply:
any of the two record being zero would mean this isn't a TRT tag

Reply:
Why not `||` ? ;)

Reply:
Ah! My bad. Fixed.